
# HG changeset patch
# User Hans de Goede <hdegoede@redhat.com>
# Date 1264955132 -3600
# Node ID bf7af07e9bbdeb453b866829107d88496bf9f329
# Parent  923ef40994edbc80c6d9308b05b90cfabee984d0
libv4l: enable whitebalance for sonixj + ov7630 cams

From: Hans de Goede <hdegoede@redhat.com>

libv4l: enable whitebalance for sonixj + ov7630 cams

Priority: normal

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--- a/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Sun Jan 31 16:52:42 2010 +0100
+++ b/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Sun Jan 31 17:25:32 2010 +0100
@@ -289,7 +289,11 @@
   { 0x0c45, 0x6005, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* TAS5110C */
   { 0x0c45, 0x6007, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* TAS5110D */
   /* sn9c105 / sn9c120 based devices (sonixj) */
+  { 0x0c45, 0x60fe, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* OV7630 */
+  { 0x0c45, 0x610e, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* OV7630 */
   { 0x0c45, 0x6128, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* OM6802 */
+  { 0x0c45, 0x612e, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* OV7630 */
+  { 0x0c45, 0x613e, 0,    NULL, NULL, V4LCONTROL_WANTS_WB }, /* OV7630 */
   /* Pac207 based devices */
   { 0x041e, 0x4028, 0,    NULL, NULL, V4LCONTROL_WANTS_WB, 1500 },
   { 0x093a, 0x2460, 0x1f, NULL, NULL, V4LCONTROL_WANTS_WB, 1500 },

